Makefile

```diff
@@ -215,27 +215,30 @@ dist/img-oci-artifact.tar: target/wasm32-wasip1/$(OPT_PROFILE)/img-oci-artifact.
 	@mkdir -p "dist/"
 	cp "$<" "$@"
 
-load: dist/img.tar
-	sudo ctr -n $(CONTAINERD_NAMESPACE) image import --all-platforms $<
+.PHONY: pull pull-app pull-oci pull-oci-artifact pull-http
+pull: pull-app pull-oci pull-oci-artifact pull-http
+	echo "Pulled all images"
+
+pull-app:
+	ctr image pull ghcr.io/containerd/runwasi/wasi-demo-app:latest
```

Comment:
For load, we do `make load`, and the makefile does `sudo ctr ..`.
This one seems to be the other way around, `sudo make pull`, while the makefile does just `ctr ..`.
Is there any reason for that? Otherwise I think I prefer the way `load` does it.

Reply:
I agree. I will change it to `sudo ctr` in the Makefile